stellargraph/data/explorer.py

```diff
@@ -796,3 +800,202 @@ def _check_neighbourhood_sizes(self, in_size, out_size):
             self._raise_error(
                 "The number of hops for the in and out neighbourhoods must be the same."
             )
+
+
+""" Temporal random walks """
+
+
+class TemporalRandomWalk(GraphWalk):
+    """
+    Performs uniform temporal random walks on the given graph
+    """
+
+    def run(
+        self,
+        nodes=None,
+        n=None,
+        length=None,
+        bidirectional=False,
+        edge_time_label="weight",
```

Comment:
We can remove `edge_time_label` and all its usage in this class entirely. This is a parameter that needs to be used when creating the graph i.e. `StellarGraph(nxg, edge_weight_label="time")` or something, not in this class.